src/content/guides/how-to-revoke-token-access/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: How to revoke smart contract access to your crypto funds
+description: A how to guide on revoking exploitative smart contract token access
+lang: en
+---
+
+# How to: Revoke smart contract access to your crypto funds
```

Comment:
Is this prefix necessary? It's not a convention we use.

```suggestion
# Revoke smart contract access to your crypto funds
```

Reply:
"how to" is a language used by users in survey responses and i think indicates mental model of a guide, i would recommend keeping it